README.md

```diff
@@ -1,2 +1,79 @@
-# plugin-template
-Template repository for plugins that will run within Ubiquibot.
+# `@ubiquibot/plugin-template`
+
+Still a work in progress while some aspects of the kernel are being actively worked on.
+
+It contains the ingredients for a basic plugin, including the necessary files and a basic structure. This is not a complete guide or fully featured template at this stage, but it should be enough to get you started. If you have any questions, pop into the Telegram chat and ask or dm me directly, I'm happy to help.
+
+## Prerequisites
+
+- A basic understanding of GitHub Actions, writing workflows, and the GitHub API.
+- A good understanding of how the kernel works and how to interact with it.
+- A basic understanding of the Ubiquibot configuration file and how to define your plugin's settings.
+
+## Getting Started
+
+1. Create a new repository using this template.
+2. Clone the repository to your local machine.
+3. Install the dependencies using your package manager of choice.
+
+## Creating a new plugin
+
+1. Ensure you understand and have setup the [kernel](https://github.com/ubiquity/ubiquibot-kernel).
+2. Update [.ubiquibot-config.yml](./.github/.ubiquibot-config.yml) with your plugin's information. This is the configuration file for _this_ repository, you should also have a private org repo `.ubiquibot-config` with `./github/.ubiquibot-config.yml` as the contents. Here is where you should store sensitive keys like `keys.openAi` and `keys.evmPrivateEncrypted`.
```

Comment:
The config comes from either and org wide, or the targetted repo, not the plugin itself.

Reply:
I think my mental model has been a little off especially when it comes to auth/secrets in terms of being a `partner` and setting up the bot and having to install plugins vs developing plugins and being setup for testing.